lib/drivers/node-mongodb-native/collection.js

```diff
@@ -208,8 +208,14 @@ function iter(i) {
 
     if (debug) {
       if (typeof debug === 'function') {
+        let range = '';
```

Comment:
Please stop initializing values that aren't strings to empty strings. If you want to add a placeholder value, do `let range = null`